polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,67 @@
+# This script includes docking related evaluation metrics.
+
+from typing import Union, List
+
+import numpy as np
+from rdkit.Chem.rdMolAlign import CalcRMS
+import datamol as dm
+
+
+## The following rmsd implementation are modified based on ppsebusters https://github.com/maabuu/posebusters/blob/main/posebusters/modules/rmsd.py
+
+
+def _rmsd(mol_probe: dm.Mol, mol_ref: dm.Mol) -> float:
```

Comment:
**Question:** Are we confident the user will know to align the molecule beforehand? I am sure that for the cheminformatics community working on Docking this is common practice, but we're targeting ML scientists here who may not have that background. If it's as easy as calling a single method and it doesn't change an already aligned prediction, maybe we should just add it in? Or could we somehow check for alignment and raise an error if not aligned and ask the user to do it?

Reply:
The alignment needs to be done on the target protein with the docked ligand.  I don't think we should include that here.  Whoever decide to work on docking related task in AI community are supposed to know that.

Nevertheless, we can clarify that in the documentation.